query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresBasedRequestHandler.java

```diff
@@ -472,7 +472,8 @@ Observable<Row> convert(ResultSet resultSet) throws SQLException {
         for (int c = 1; c <= columnCount; c++) {
           String colVal = resultSet.getString(c);
```

Comment:
Do we treat every result as a string? is that consistent with what we do in pinot?

Reply:
Sounds like yes